template/cmd_main.tmpl

```diff
@@ -1,16 +1,27 @@
 package main
 
 import (
+	"runtime"
+
 	"github.com/lileio/lile"
 	"github.com/lileio/lile/fromenv"
 	"github.com/lileio/pubsub"
 	"{{ .RelDir }}"
 	"{{ .RelDir }}/server"
 	"{{ .RelDir }}/{{ .SnakeCaseName }}/cmd"
+	"github.com/mattn/go-colorable"
+	"github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
 )
 
 func main() {
+	if runtime.GOOS == "windows" {
```

Comment:
There is some work elsewhere to auto insert things like this via the generator and flags, would that be ok?

i.e by default this doesn't exist, but you can `lile new github.com/lile/someservice --windows` and it will insert these things?

Reply:
I'm also going to abstract the logger soon, behind an interface, so we can do that in the logrus logger package?

Reply:
infact, can you make this a seperate PR and I'll happily accept the GOPATH fixes

Reply:
okay, will do soon.